bucket/jim.json

```
@@ -0,0 +1,13 @@
+{
+    "version": "0.9.9.9.7",
+    "description": "TF2 Item Manager",
+    "url": "http://www.jengerer.com/item_manager/downloads/item_manager.zip",
+    "homepage": "http://www.jengerer.com/item_manager/",
+    "bin": "item_manager.exe",
+    "shortcuts": [
+        [
+            "item_manager.exe",
+            "Jengerer's Item Manager"
+        ]
+    ]
```

According to its homepage, the Visual C++ 2010 redistributable is required to run the program:

```suggestion
    ],
    "depends": "extras/vcredist2010"
```